src/components/Banner.tsx

```diff
@@ -0,0 +1,97 @@
+import {
+  Collapse,
+  Alert,
+  Stack,
+  Typography,
+  Divider,
+  IconButton,
+  Button,
+  Box,
+  SxProps,
+  Theme,
+} from "@mui/material";
+import React, {useState} from "react";
+import CloseIcon from "@mui/icons-material/Close";
+import ArrowForwardIosIcon from "@mui/icons-material/ArrowForwardIos";
+import {grey} from "../themes/colors/aptosColorPalette";
+import AptosBannerImage from "../assets/Banner.jpg";
+
+interface BannerProps {
+  children: React.ReactNode;
+  handleClick: () => void;
+  sx?: SxProps<Theme>;
+}
+
+export function Banner({children, handleClick, sx}: BannerProps) {
+  const [bannerOpen, setBannerOpen] = useState<boolean>(true);
+
+  return (
+    <Collapse in={bannerOpen}>
+      <Box
+        sx={[
+          {
+            textOverflow: "ellipsis",
+            overflow: "hidden",
+            whiteSpace: "nowrap",
+          },
+          ...(Array.isArray(sx) ? sx : [sx]),
+        ]}
+      >
+        <Alert
+          sx={{backgroundImage: `url(${AptosBannerImage})`, borderRadius: 1}}
+          icon={false}
+          action={
+            <Stack
+              direction="row"
+              spacing={1}
+              marginRight={2}
+              sx={{verticalAlign: "center"}}
+            >
+              <Button variant="text" onClick={handleClick}>
```

Comment:
what does this button do. does that mean it shows the same thing when clicking the button and clicking the banner itself?

Reply:
right now it doesn't do anything, cuz we don't have the side panel created yet. but basically once side panel's added, when users click on this button, it will open up side panel with delegation FAQs.